nifi-assembly/pom.xml

```diff
@@ -760,6 +760,12 @@ language governing permissions and limitations under the License. -->
             <version>1.15.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-hashicorp-vault-nar</artifactId>
```

Comment:
What do you think of naming this to something more specific? It seems like there could be other libraries for interacting with HashiCorp Vault, so including `parameter-value-provider` in the name would be helpful.

Reply:
My thought was that this nar would also contain jars for a `hashicorp-vault-parameter-provider` and a `hashicorp-vault-controller-service`.  In light of that, I was considering this more analogous to the `nifi-aws-nar`.  What do you think?

Reply:
Thanks for the reply, that seems reasonable, and it could always separated later if needed.